src/main/java/DZ/examination/Examination.java

```diff
@@ -0,0 +1,12 @@
+package DZ.examination;
+
+public class Examination {
+    public static boolean isDigit(String s) throws NumberFormatException {
+        try {
+            Integer.parseInt(s);

```

Comment:
If you use this only to chech, if input is dgit, you can use method from Apache Commons library